lib/account/pages/edit_account/edit_birth_date_page.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/material.dart';
+import 'package:motis_mitfahr_app/util/locale_manager.dart';
+import 'package:motis_mitfahr_app/util/supabase.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+import '../../../util/big_button.dart';
+import '../../models/profile.dart';
+
+class EditBirthDatePage extends StatefulWidget {
+  final Profile profile;
+  const EditBirthDatePage(this.profile, {super.key});
+
+  @override
+  State<EditBirthDatePage> createState() => _EditBirthDatePageState();
+}
+
+class _EditBirthDatePageState extends State<EditBirthDatePage> {
+  final TextEditingController _controller = TextEditingController();
+  DateTime? _date;
+
+  @override
+  void initState() {
+    super.initState();
+    _date = widget.profile.birthDate;
+    _controller.text = _date == null ? '' : localeManager.formatDate(_date!);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(S.of(context).pageProfileEditBirthDateTitle),
+      ),
+      body: Padding(
+        padding: const EdgeInsets.symmetric(horizontal: 8, vertical: 10),
+        child: Column(
+          children: [
+            TextField(
+              decoration: InputDecoration(
+                border: const OutlineInputBorder(),
+                hintText: S.of(context).pageProfileEditBirthDateHint,
+                suffixIcon: _getClearButton(),
+              ),
+              onTap: _showDatePicker,
+              controller: _controller,
+            ),
+            const SizedBox(
+              height: 10,
+            ),
+            BigButton(
+              onPressed: onPressed,
+              text: S.of(context).save,
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Widget? _getClearButton() {
+    if (_controller.text == '') {
+      return null;
+    }
+    return IconButton(
+      icon: const Icon(Icons.clear),
+      onPressed: () {
+        _controller.clear();
+        _date = null;
+      },
+    );
+  }
+
+  void _showDatePicker() async {
+    final twelveYearsAgo = DateTime.now().subtract(const Duration(days: 365 * 12));
+    DateTime? date = await showDatePicker(
```

Comment:
can we use another DatePicker here or that you can just type the date. I think it's a little bit inconvenient since you have to change the Year and Month of the DatePicker. We could for Example use de Cupertino DatePicker there you can just scroll through the Years, Month but not sure if that works for Android

Reply:
There is actually a button within the Date picker that lets you type it in. I think we should always use the picker of the target platform, as an Android user I would find an iOS picker really weird. (To be fair, there are far better pickers used in a lot of Android Apps, I would argue in favor of using a package to replace the current picker in another PR)

Reply:
sounds good.